Source/Renderer/DrawCommand.js

```diff
@@ -32,6 +32,17 @@ define([
         this.boundingVolume = options.boundingVolume;
 
         /**
+         * The oriented bounding box of the geometry in world space. If this is defined, it is used instead of
+         * {@link DrawCommand#boundingVolume} for plane intersection testing.
+         *
+         * @type {OrientedBoundingBox}
+         * @default undefined
+         *
+         * @see DrawCommand#debugShowBoundingVolume
+         */
+        this.boundingOBB = options.boundingOBB;
```

Comment:
Since these are attached to the DrawCommand, there are other places in the code where it doesn't work, e.g. in `Scene.js` `createPotentiallyVisibleSet`:

```
BoundingSphere.computePlaneDistances(boundingVolume, position, direction, distances);
```

I could implement `computePlaneDistances` for `OrientedBoundingBox`, but I'm not clear on whether we really want `createPotentiallyVisibleSet` to be calling that.